GitPrompt.ps1

```diff
@@ -82,6 +82,9 @@ $global:GitPromptSettings = New-Object PSObject -Property @{
 
     AutoRefreshIndex                            = $true
 
+    # Valid values are "Full", "Compact", and "Minimal"
+    BranchBehindAndAheadDisplay                 = "Full"

```

Comment:
Not sure about defaulting to `Full` - we're already taking up more space than we used to. Then again, leaving the counts disabled by default means most people will never see them. Anyone else have an opinion?